data/mil2525d/utilities/legacy-stylx-utilities/Sqlite.sql

```diff
@@ -0,0 +1,16 @@
+.mode csv
+/* TODO: UPDATE PATH BELOW TO PATH ON YOUR LOCAL MACHINE - ONLY FORWARD SLASHES '/' IN PATH - NOT BACKSLASHES '\' */
+.import "C:/{TODO UPDATE PATH}" LegacyMapping
+
+CREATE INDEX 'LEGACYMAPPING_KEY2525C' ON 'LegacyMapping' ('Key2525C');
+
+/* WORKAROUND TO GET Special C2 Entity Icons to show up until fixed in source data */
+UPDATE LEGACYMAPPING SET ExtraIcon = '10XXXX97' WHERE MainIcon like '%97';
```

Comment:
@abouffard @Dbarnes1  - should this workaround be here anymore? If it is no longer needed - it should be removed.

I believe this was fixed in the data: https://github.com/Esri/joint-military-symbology-xml/blob/master/samples/legacy_support/All_ID_Mapping_C_to_D.csv